packages/generator/src/parser/parser-util.ts

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+
+export function stripNamespace(name: string): string {
+  const nameParts = name.split('.');
+  return nameParts[nameParts.length - 1];
+}
+
+const collectionRegExp = /Collection\((?<collectionType>.*)\)/;
```

Comment:
In the API v4 edmx I find the following collection type: `Collection(API_TEST_SRV.A_TestComplexType)`. This would mean the regex would take: `API_TEST_SRV.A_TestComplexType` right? But the namespace in front should be part of the type? Is this tested?

Reply:
Good point, I didn' test this directly. I will add some unit tests.